### PR TITLE
Examples of some basic python control structures-conditions

### PR DESCRIPTION
**Pull request by imahnama**

### CODE REVIEW

**Comment by bqevin** [comment]
[Conditions](https://trello.com/c/3Qd0KVoP/4-conditions)

File: conditions.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# if BOOLEAN EXPRESSION:
+#     STATEMENTS
+food ='chapati'
+if food == 'chapati':
+    print('My favourite!')
+
+# if-else STATEMENT
+
+food ='chips'
+if food == 'chapati':
```
Comment (bqevin) [inline conditions.py]: You can do strict typing comparison with 3 equal signs i.e `===`

File: conditions.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# # general format =
+#                     for LOOP_VARIABLE in SEQUENCE:
+#                         STATEMENTS
+
+for friend in ['Nama', 'Cate', 'Sue']:
+    invitation = "Hi" + friend + ". Please come to my party on Sat!"
+    print(invitation)
+
+# BOOLEAN  DATA OBJECTS
+print(5==10)
```
Comment (bqevin) [inline conditions.py]: Same here to compare with strict type

File: conditions.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# While - repeats a body of code as long as condition is true
+
+counter = 1
+while counter <= 5:
+    print("Hello, World")
+    counter = counter + 1
+
+# if BOOLEAN EXPRESSION:
+#     STATEMENTS
+food ='chapati'
```
Comment (bqevin) [inline conditions.py]: For strings let's try to always use double quotes unless otherwise

File: conditions.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+if food == 'chapati':
```
Comment (bqevin) [inline conditions.py]: Double quotes for strings

File: conditions.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    print('My favourite!')
```
Comment (bqevin) [inline conditions.py]: Double quotes for strings

**Comment by bqevin** [comment]
Otherwise really good use of comment blocks!

File: conditions.py (hunk 2 above)
Comment (bqevin) [inline conditions.py]: Check the code style...space before the operators and after them i.e `5 == 10`

File: conditions.py (hunk 5 above)
Comment (imahnama) [reply conditions.py]: Noted

File: conditions.py (hunk 1 above)
Comment (imahnama) [reply conditions.py]: Noted

**Pull request merged**